internal/acceptance/rekor/rekor.go

```diff
@@ -66,7 +66,31 @@ func stubRekordRunning(ctx context.Context) (context.Context, error) {
 		return ctx, err
 	}
 
-	return wiremock.StartWiremock(ctx)
+	if state.KeyPair == nil {
+		// not used for any signing, we just need the public key in PEM for the
+		// in-toto schema below
+		keyPair, err := crypto.GenerateKeyPair()
+		if err != nil {
+			return ctx, err
+		}
+
+		state.KeyPair = keyPair
+	}
+
+	ctx, err = wiremock.StartWiremock(ctx)
+	if err != nil {
+		return ctx, err
+	}
+
+	if err = wiremock.StubFor(ctx, wiremock.Get(wiremock.URLPathEqualTo("/api/v1/log/publicKey")).
```

Comment:
Something for the future... In a rekor installation, is the public key auto-generated?

Reply:
Few lines up in that method.

Reply:
I see how it's done here for the acceptance tests. I meant on a real installation.

Reply:
That would be something that Rekor does on it's on. From what I understand it can either [generate the key and hold it in-memory](https://github.com/sigstore/rekor/blob/07ec8fc064309472d62c939c0e04b7b56fac15b7/pkg/signer/memory.go#L36), or use a [GCP KMS reference](https://github.com/sigstore/rekor/blob/07ec8fc064309472d62c939c0e04b7b56fac15b7/pkg/signer/signer.go#L30-L31) (configured via `--rekor_server.signer`).